src/lib.rs

```diff
@@ -247,8 +209,9 @@ struct PlayerBundle {
 
 #[derive(Component, Deserialize, Serialize)]
 struct Player;
-# fn serialize_transform(_: Ptr, _: &mut Cursor<Vec<u8>>) -> bincode::Result<()> { unimplemented!() }
-# fn deserialize_transform(_: &mut EntityWorldMut, _: &mut ServerEntityMap, _: &mut Cursor<&[u8]>, _: RepliconTick) -> bincode::Result<()> { unimplemented!() }
+# /// To avoid enabling `serialize` feature on Bevy.
+# #[derive(Component, Deserialize, Serialize)]
+# struct Transform;
```

Comment:
It will work for users, they just need to enable `serialize` feature, without it no components will implement `serialize`.

But I just realized that I can enable this feature in `dev-dependencies`.